.github/workflows/frontend-docker-publish-image.yml

```diff
@@ -63,6 +63,12 @@ jobs:
                   echo ${GITHUB_SHA} > CI_COMMIT_SHA
                   echo ${{ steps.meta-build.outputs.tags }} > IMAGE_TAG
 
+            - name: Download features
+              run: >
+                curl --location --request GET 'https://api.flagsmith.com/api/v1/flags/'
+                --header 'X-Environment-Key: ${{ secrets.ENVIRONMENT_KEY }}' > flags.json
```

Comment:
We need to do this in the /frontend directory
Also can we call it `FLAGSMITH_ENVIRONMENT_KEY`